test_charts.sh

```
@@ -17,6 +19,13 @@ function testChart() {
         echo "Unknown field detected"
         exit 1
     fi
+
+    # If chart has a test folder, run `task test`
+    if [ -d "test" ]; then
+        cd test
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)